companies/intel.md

```diff
@@ -2,7 +2,7 @@
 fortune_rank: 45
 name: Intel
 tags: ["fortune500", "tech", "computers", "artificial intelligence"]
-image: name-in-images-folder.png
+image: EZe6DCEWoAEim0h.jpeg
```

Comment:
Nice! Do you mind renaming the image to `intel.jpeg`?

Reply:
no problem, it's done!